inc/Engine/Admin/DomainChange/Subscriber.php

```diff
@@ -63,6 +64,10 @@ public static function get_subscribed_events() {
 	 * @return void
 	 */
 	public function maybe_launch_domain_changed() {
+		if ( wp_doing_ajax() ) {
```

Comment:
are we sure this is going to be enough for all i18n plugins we're compatible with?

Reply:
@MathieuLamiot I'd release it asap and then go with more hotfixes versions if needed. We knew that this update will be tricky but it's not possible to do exploratory testing for whole WP ecosystem.

We can do a quick tests for some language switchers of multilingual plugins until it passes the CR.

Reply:
@CrochetFeve0251, @engahmeds3ed, @Tabrisrp can we move forward with the CR quickly then, so that @Mai-Saad can do QA?
Thank you 🙏